test/features/support_archive/support_archive.go

```diff
@@ -187,7 +187,7 @@ func executeSupportArchiveCommand(ctx context.Context, t *testing.T, envConfig *
 	return executionResult
 }
 
-func assertFile(t *testing.T, requiredFiles []string, zipFile zip.File) []string {
+func assertFileSize(t *testing.T, requiredFiles []string, zipFile zip.File) []string {
```

Comment:
This function is just weird, and the rename didn't make it better 😅 
- it checks if the "file is not empty" -> `assertFileSize` reads to me that there is a specific none-zero size we wish to assert for
- `assert...` name is already misleading as the "asserting" part is not it's main/only job, this function will filter out the present files from the `requiredFiles` so later the not-present `requiredFiles` can be logged (and fail the test).

Reply:
correct, but i don't have better name, and in general I prefer to refactor it.

i'll revert change, and we can discuss how to refactor this part.